pkgs/by-name/ve/verovio/package.nix

```diff
@@ -0,0 +1,68 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  cmake,
+  git,
+  makeWrapper,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "verovio";
+  version = "4.2.0";
+
+  src = fetchFromGitHub {
+    owner = "rism-digital";
+    repo = finalAttrs.pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21